src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/FallbackRazorConfiguration.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.Language;
+
+namespace Microsoft.CodeAnalysis.Razor.ProjectSystem
+{
+    internal class FallbackRazorConfiguration : RazorConfiguration
```

Comment:
Moving this up a level simplifies a lot of things. Now we have a static list of supported legacy configurations. In addition since this is a different type now, we can tell at a glance if the configuration come from the project system or if it was a fallback.

Reply:
How is it simpler than just being in Mvc.Razor.Extensions?

Reply:
These assemblies aren't going to reference that when I'm done 👍

Reply:
I should mention that supports introduces loose coupling based on the configuration/extension names. So VS will be able to find `MVC-1.1` and configure it, but without anything in between having a direct reference.

The change here is that the workspaces assembly reflects our history. It hardcodes the notion of MVC's old configurations but doesn't hardcode the implementation.

Reply:
I see I see I see. Ok 👍